### PR TITLE
BO: Fix shipping price tax issue on order creation page

### LINKED ISSUE #17773: Create order : Shipping is tax incl instead of excl
**Describe the bug**
On the new migrated create order page (/sell/orders/orders/new), in the summary of the order, the "Total shipping" is displayed tax incl., but according to the label (and to Legacy page), it should be tax excl.

**To Reproduce**
1. Go to Create Order migrated page
2. Add a product, and a shipping (not free)
3. See Summary : Shipping is displayed tax incl.

**Screenshots**
create order shipping excl

**Additional information**
PrestaShop version: 1.7.7
PHP version: N/A

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | On order creation page, the total shipping price must be displayed without taxes, not tax included.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #17773
| How to test?  | Steps to reproduce the behavior:<br><br>- Go to Create Order migrated page<br>- Add a product, and a shipping (not free)<br>- See Summary : Shipping must be displayed without tax (tax exl.)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17853)

### CODE REVIEW

**Comment by matthieu-rolland** [comment]
To sum up, this PR does two things:

1/ Add a new **totalShippingWithoutTaxes** attribute to the CartSummary class. (there was only the total shipping price tax included before that).

2/ Use that value in create order page, in the order summary.

**Review by matks — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by nesrineabdmouleh** [comment]
Hello @matthieu-rolland,

See Shipping : Shipping is displayed tax excl.
image

Thanks,

**Comment by matthieu-rolland** [comment]
> Hello @matthieu-rolland,
> 
> See Shipping : Shipping is displayed tax excl.
> image
> 
> Thanks,

thank you @nesrineabdmouleh ! it should be good now 👍

File: admin-dev/themes/new-theme/js/pages/order/create/summary-renderer.js [hunk 1]
```diff
@@ -61,9 +62,10 @@ export default class SummaryRenderer {
       return;
     }
     const cartSummary = cartInfo.summary;
-
+    console.log(cartSummary);
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/order/create/summary-renderer.js]: You pushed too fast 😄

**Review by matks — CHANGES_REQUESTED** [review]
See comment, `console.log()` to remove

**Review by Progi1984 — COMMENTED** [review]
OK. But look at @matks comment.

**Review by matks — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Thanks @matthieu-rolland :)

**Comment by Progi1984** [comment]
Thanks @matthieu-rolland

**Pull request merged**